styles/logo.scss

```diff
@@ -0,0 +1,5 @@
+.c-logo {
+	background: url("../../svg/logo.svg");
```

Comment:
the path is wrong. Your project starts from this folder, it means that all needed files should be located inside this folder:
![image](https://github.com/tacya9/netfix/assets/115877101/5dc1b283-0d8b-4a58-820f-ef72f7960a98)